docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -220,3 +253,14 @@ binary and algorithmic compatibility and some other steps.
 
 It is worth mentioning that a business app upgrade is orthogonal to a
 rolling member upgrade. A rolling business app upgrade may be done without upgrading the members.
+
+**Are features marked as Beta in my current version upgraded?**
+
+Features marked as Beta are not upgraded. This is due to changes that can occur between Beta and general availability releases. If a Beta feature has been updated or made generally available in the target release, that version of the feature is included in the upgraded feature set.
+
+**Can I upgrade directly to the latest release?**
+
+You can upgrade directly to the latest release in the following circumstances:
+
+* Your current version is no more than one minor version below the current release
+* The latest release is an xref:migrate:lts.adoc[LTS release] and your current version is among those supported
```

Comment:
yes, and fyi the current supported version will be only the previous LTS and the previous minor in the future. For now it's 5.2-5.4 for 5.5.

Reply:
I am a little confused with `current version` and `current release`. Is it going to be better if `source version` and `target version` are used?

Reply:
Suggesting using `source` and `target` wherever possible. e.g. Your source version is no more than one minor version below the current release (target)

Reply:
Amended